packages/foreman/foreman/dynflow@.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
```

Comment:
Can we ship this file in the foreman.git repo? We do this for smart-proxy (https://github.com/theforeman/smart-proxy/blob/develop/extra/systemd) and foreman (https://github.com/theforeman/foreman/tree/develop/extras/systemd) and it has the benefit of being able to share configs between debs and rpms.